addons/ai/fnc_taskDefend.sqf

```diff
@@ -38,13 +38,12 @@ _position = [_position,_group] select (_position isEqualTo []);
 _position = _position call CBA_fnc_getPos;
 
 [_group] call CBA_fnc_clearWaypoints;
-_group enableAttack false;
```

Comment:
Since it's now using `_unit disableAI "PATH";` this allows units not assigned to a position to counterattack in the case of contact